test/components/util/util.go

```diff
@@ -88,14 +90,15 @@ func CreateKubeClient(t *testing.T) *kubernetes.Clientset {
 }
 
 func WaitForStatefulSet(t *testing.T, client kubernetes.Interface, ns, name string, replicas int, retryInterval, timeout time.Duration) {
-	if err := wait.Poll(retryInterval, timeout, func() (done bool, err error) {
+	if err := wait.PollImmediate(retryInterval, timeout, func() (done bool, err error) {
 		ds, err := client.AppsV1().StatefulSets(ns).Get(context.TODO(), name, metav1.GetOptions{})
 		if err != nil {
 			if k8serrors.IsNotFound(err) {
 				t.Logf("waiting for statefulset %s to be available", name)
 				return false, nil
 			}
-			return false, err
+
+			return false, fmt.Errorf("getting StatefulSet %q: %w", name, err)
```

Comment:
Just wanting to confirm, shouldn't it be `error getting StatefulSet %q: %w`. I may have not understood the full context behind it, but happy to know if you can explain it.

Reply:
Is it becasue of the decorated error pattern?

Reply:
I'm following https://github.com/kinvolk/lokomotive/issues/59#issuecomment-648894086 here, to avoid putting unnecessary words in error messages.